recipes/ndcube/bld.bat

```diff
@@ -0,0 +1,2 @@
+"%PYTHON%" setup.py install --offline --no-git --single-version-externally-managed --record record.txt
```

Comment:
I don't think you need the `bld.bat` if you have the `script` entry in the `meta.yaml`